OmniBLE/PumpManagerUI/Views/DeliveryUncertaintyRecoveryView.swift

```diff
@@ -19,7 +19,7 @@ struct DeliveryUncertaintyRecoveryView: View {
 
     var body: some View {
         GuidePage(content: {
-            Text(String(format: LocalizedString("%1$@ has been unable to communicate with the pod on your body since %2$@.\n\nWithout communication with the pod, the app cannot continue to send commands for insulin delivery or display accurate, recent information about your active insulin or the insulin being delivered by the Pod.\n\nMonitor your glucose closely for the next 6 or more hours, as there may or may not be insulin actively working in your body that %3$@ cannot display.", comment: "Format string for main text of delivery uncertainty recovery page. (1: app name)(2: date of command)(3: app name)"), self.model.appName, self.uncertaintyDateLocalizedString, self.model.appName))
+            Text(String(format: LocalizedString("%1$@ has been unable to communicate with the pod on your body since %2$@.\n\nWithout communication with the pod, the app cannot resolve information about your active insulin or the insulin being delivered by the Pod. If you cannot resolve communication with this pod, you should monitor your glucose closely for the next 6 or more hours, as there may or may not be insulin actively working in your body that %3$@ cannot display.\n\nThe app will keep trying to restore communication. If you decide to give up, the Deactive Pod action will probably fail and you can discard the pod and start a new one. If it succeeds, you should have waited longer.", comment: "Format string for main text of delivery uncertainty recovery page. (1: app name)(2: date of command)(3: app name)"), self.model.appName, self.uncertaintyDateLocalizedString, self.model.appName))
```

Comment:
Typo here - `Deactive Pod` should be `Deactivate Pod`.

Also it seems it would be a good idea to provide some additional guidance here - what is the minimum amount of time someone should wait? I think the statement "you should have waited longer" isn't helpful... to me it speaks to not actually wanting to deactivate the pod - rather it is "disconnect from pod" or "unpair pod" (and as part of that it would check the comms with the pod one last time and only disconnect if it fails).